content/docs/deployment/csminstallationwizard/src/static/js/utility.js

```diff
@@ -84,6 +87,15 @@ function validateForm(CONSTANTS_PARAM) {
 			}
 		}
 	}
+	if (powerflexSelected) {
+		if (document.getElementById('installation-type').value === CONSTANTS_PARAM.HELM) {
+			if (renameSDCEnabled) {
+				if (document.getElementById('sdc-prefix').value.trim() === "") {
```

Comment:
We can also add this 4 conditions with and `&&` operator, instead of nested if.

Reply:
hi @samihan-dell , its done in nested way to keep it in sync as like PowerMax has done it previously, also makes it more readable as well as in future if any other key is to be validated, it can be added accordingly.